src/main/java/com/hedvig/backoffice/services/underwriter/UnderwriterClient.kt

```diff
@@ -40,9 +36,17 @@ interface UnderwriterClient {
   @PostMapping("/_/v1/quotes/{quoteId}/activate")
   fun activateQuote(@PathVariable("quoteId") quoteId: UUID, @RequestBody body: ActivateQuoteRequestDto): QuoteDto
 
+  @PostMapping("/_/v1/quotes/add/agreement")
+  fun addAgreementFromQuote(@RequestBody request: AddAgreementFromQuoteRequest): QuoteDto
+
   @GetMapping("/_/v1/quotes/members/{memberId}")
   fun getQuotes(@PathVariable("memberId") memberId: String): List<QuoteDto>
 
   @GetMapping("/_/v1/quotes/{quoteId}")
   fun getQuote(@PathVariable("quoteId") quoteId: UUID): QuoteDto
+
+  @PostMapping("/_/v1/quotes/createQuoteFromAgreement")
```

Comment:
Very subjective but I like endpoints that look like this
```suggestion
  @PostMapping("/_/v1/quotes/create/from/agreement")
```